setup.cfg

```diff
@@ -11,7 +11,7 @@ addopts =
     --ignore examples
     --ignore maint_tools
     --doctest-modules
-    --disable-pytest-warnings
+    # --disable-pytest-warnings
```

Comment:
I suppose this is temporary to check that there are no more deprecation warnings?

Reply:
yes or that I am removing what I am supposed to do when removing the filter as well